sky/backends/backend_utils.py

```diff
@@ -341,7 +341,10 @@ def path_size_megabytes(path: str) -> int:
                      f'{git_exclude_filter} --dry-run {path!r}')
     rsync_output = ''
     try:
-        rsync_output = str(subprocess.check_output(rsync_command, shell=True))
+        rsync_output = str(
+            subprocess.check_output(rsync_command,
+                                    shell=True,
+                                    stderr=subprocess.DEVNULL))
```

Comment:
We should leave a comment here for why we set the `stderr` to NULL and why it is ok